docker/utils/utils.py

```diff
@@ -509,6 +509,13 @@ def create_host_config(
     if devices:
         host_config['Devices'] = parse_devices(devices)
 
+    if group_add:
+        if compare_version(version, '1.20') < 0:
```

Comment:
I think this should have been `>`, not `<` - if I set the DEFAULT_DOCKER_API_VERSION to '1.21', then this error gets raised.